### PR TITLE
FAQ cleanup

### PR DESCRIPTION
**Pull request by foss-**
- updated client section
- updated contact states section

### CODE REVIEW

File: doc/FAQ.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ User
 * **[How can I upload images, files, links, videos and sound files to posts?](help/FAQ#upload)**
 * **[Is it possible to have different avatars per profile?](help/FAQ#avatars)**
 * **[How can I view Friendica in a certain language?](help/FAQ#language)**
-* **[What is the difference between blocked|ignored|archived|hidden contacts?](help/FAQ#contacts)**
+* **[How do blocked, ignored, archived and hidden contacts behave?](help/FAQ#contacts)**
 * **[What happens when an account is removed? Is it truly deleted?](help/FAQ#removed)**
 * **[Can I subscribe to a hashtag?](help/FAQ#hashtag)**
 * **[How to create a RSS feed of the stream?](help/FAQ#rss)**
-* **[Are there any clients for friendica I can use?](help/FAQ#clients)**
+* **[List of friendica clients](help/FAQ#clients)**
```
Comment (MrPetovan) [inline doc/FAQ.md]: Why did you change this from a question to a headline even though it's the style we've been using for the FAQ?

File: doc/FAQ.md [hunk 2]
```diff
@@ -94,26 +94,25 @@ in German:
...
-They are not included in delivery, and their own posts to you are not imported.
-However their conversations with your friends will still be visible in your stream.
-If you remove a contact completely, they can send you another friend request.
-Blocked contacts cannot do this. They cannot communicate with you directly, only through friends.
+##### Blocked
 
-**Ignored contacts** are included in delivery - they will receive your posts and private messages.
-However we do not import their posts or private messages to you.
-Like blocking, you will still see this person's comments to posts made by your friends.
+Direct communication will be blocked. Blocked contacts are not included in delivery, and their own posts to you are not imported. However their conversations with your friends will still be visible in your stream. If you remove a contact completely, they can send you another friend request. Blocked contacts cannot do this. They cannot communicate with you directly, only through friends.
```
Comment (MrPetovan) [inline doc/FAQ.md]: Please add a new line after each sentence. It doesn't affect the display but makes this file easier to read and change. This goes for all Markdown files.

File: doc/FAQ.md [hunk 3]
```diff
@@ -94,26 +94,25 @@ in German:
...
-An **archived contact** means that communication is not possible and will not be attempted.
-(Perhaps the person moved to a new site and removed the old profile.)
-However unlike blocking, existing posts this person made before being archived will be visible in your stream.
+Not included in delivery but they will receive your posts and private messages. However we do not import their posts or private messages to you. Like blocking, you will still see this person's comments to posts made by your friends.
 
-A **hidden contact** will not be displayed in any "friend list" (except to you).
-However a hidden contact will appear normally in conversations and this may expose his/her hidden status to anybody who can see the conversation.
+An addon called "blockem" can be installed to collapse/hide all posts from a particular person in your stream if you desire complete blocking of an individual, including their conversations with your other friends.
+
+##### Archieved
```
Comment (MrPetovan) [inline doc/FAQ.md]: Typo
```suggestion
##### Archived
```

File: doc/FAQ.md [hunk 4]
```diff
@@ -94,26 +94,25 @@ in German:
...
+Direct communication will be blocked. Blocked contacts are not included in delivery, and their own posts to you are not imported. However their conversations with your friends will still be visible in your stream. If you remove a contact completely, they can send you another friend request. Blocked contacts cannot do this. They cannot communicate with you directly, only through friends.
 
-An addon called "blockem" can be installed to collapse/hide all posts from a particular person in your stream if you desire complete blocking of an individual, including his/her conversations with your other friends.
+##### Ignored
 
-An **archived contact** means that communication is not possible and will not be attempted.
-(Perhaps the person moved to a new site and removed the old profile.)
-However unlike blocking, existing posts this person made before being archived will be visible in your stream.
+Not included in delivery but they will receive your posts and private messages. However we do not import their posts or private messages to you. Like blocking, you will still see this person's comments to posts made by your friends.
```
Comment (MrPetovan) [inline doc/FAQ.md]: "Not included in delivery but they will receive your posts and private messages."

What delivery are they not included in?

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: doc/FAQ.md (hunk 1 above)
Comment (foss-) [reply doc/FAQ.md:16]: Thought it to be more compact and quicker to grasp by users. If you want to stick with the question format, we can revert the client list title.

**Review by MrPetovan — APPROVED** [review]
